meta-mender-core/recipes-bsp/u-boot/u-boot-mender-common.inc

```diff
@@ -399,10 +421,10 @@ mender_create_fw_env_config_file_mender-ubi() {
 EOF
 }
 
-# This should evaluate to the same as MENDER_RESERVED_SPACE_BOOTLOADER_DATA.
+# This should evaluate to the same as MENDER_RESERVED_SPACE_BOOTLOADER_DATA_CHECK.
 # The only reason it's not evaluated the same way is that we don't have the
 # necessary information (BOOTENV_SIZE) when evaluating
-# MENDER_RESERVED_SPACE_BOOTLOADER_DATA.
+# MENDER_RESERVED_SPACE_BOOTLOADER_DATA_CHECK.
```

Comment:
Revert to `MENDER_RESERVED_SPACE_BOOTLOADER_DATA`, and add a comment about `MENDER_UBOOT_CONFIG_SYS_MMC_ENV_PART == 0` here as well.

Reply:
Addressed in [force push](https://github.com/mendersoftware/meta-mender/compare/f92f8114ca70418350489e978f3ae198f578d045..f65ae24861b63bcd85e22fb9fa08236339d9a801).